components/src/molecules/marketing-bar/MarketingBar.js

```diff
@@ -1,33 +1,40 @@
+// @vue/component
 import AIcon from '../../atoms/icon/Icon.vue'
 import AButton from '../../atoms/button/Button.vue'
 
-// @vue/component
 export default {
   components: {
     AIcon,
     AButton
   },
   props: {
+    /**
+     * Storage name
+     */
     dataType: {
       type: String,
       default: 'marketing-bar'
+    },
+    isBarVisible: {
```

Comment:
Missing comment.
Anyway is this prop necessary? It can be based on localstorage only I think